wsdl/ver10/device/wsdl/devicemgmt.wsdl

```diff
@@ -2159,6 +2159,11 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 							<xs:documentation>User credential for the storage server</xs:documentation>
 						</xs:annotation>
 					</xs:element>
+					<xs:element name="CertPathValidationPolicyID" type="xs:string" minOccurs="0">
```

Comment:
Better add this under extension to keep backward compatibility.

Reply:
Adjusted according to what we discussed during telco